packages/ffe-form-react/package.json

```diff
@@ -33,16 +33,16 @@
     "eslint": "^4.13.1",
     "jest": "^22.4.0",
     "prop-types": "^15.6.0",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
+    "react": "^16.4.0",
+    "react-dom": "^16.4.0",
     "react-motion": "^0.5.2",
     "react-test-renderer": "^16.2.0",
     "sinon": "^4.2.1"
   },
   "peerDependencies": {
     "@sb1/ffe-core": "^12.0.0",
     "@sb1/ffe-form": "^9.0.0",
-    "react": "^16.2.0"
+    "react": "^16.3.0"
```

Comment:
This should mirror the `devDependencies` version since consumers will actually need `16.4.x`